api/src/main/java/io/strimzi/api/kafka/model/template/KafkaMirrorMakerTemplate.java

```diff
@@ -65,6 +66,16 @@ public void setPodDisruptionBudget(PodDisruptionBudgetTemplate podDisruptionBudg
         this.podDisruptionBudget = podDisruptionBudget;
     }
 
+    @Description("Template for Kafka Mirror Maker `Deployment`.")
```

Comment:
Shouldn't we be consistent?

```suggestion
    @Description("Template for Kafka Mirror Maker container.")
```